dragonchain/lib/faas.py

```diff
@@ -0,0 +1,42 @@
+import os
+import base64
+import json
+from typing import List, Dict, Optional, cast, Any
+
+import requests
+
+from dragonchain.lib import faas
+from dragonchain import exceptions
+
+FAAS_GATEWAY = os.environ["FAAS_GATEWAY"]
+
+
+def get_faas_auth() -> str:
+    """Gets authorization to use OpenFaaS
+
+        Returns:
+            A string containing authorization for OpenFaaS.
+    """
+    with open("/etc/openfaas-secret/user", "r") as file:
+        username = file.read()
+    with open("/etc/openfaas-secret/password", "r") as file:
+        password = file.read()
+
+    return f"Basic {base64.b64encode(f'{username}:{password}'.encode('utf-8')).decode('ascii')}"
+
+
+def get_raw_logs(contract_id: str, since: Optional[str] = None, tail: Optional[int] = 100) -> List[str]:
```

Comment:
is there a reason to have a `get_logs` as well as a `get_raw_logs`? It looks like you're only using `get_logs`. Could be confusing to keep both and know which one to use in the future. Is there any reason why we'll ever want to use `get_raw_logs` separately?

Reply:
I just did it to separate the call from the transforming logic. I'll mark it as private with the leading `_`